mmv1/templates/validator/resource_converter_iam.go.erb

```diff
@@ -71,15 +71,15 @@ func resourceConverter<%= resource_name -%>IamMember() ResourceConverter {
     }
 }
 
-func Get<%= resource_name -%>IamPolicyCaiObject(d TerraformResourceData, config *transport_tpg.Config) ([]Asset, error) {
+func Get<%= resource_name -%>IamPolicyCaiObject(d tpgresource.TerraformResourceData, config *transport_tpg.Config) ([]Asset, error) {
```

Comment:
you'll need to add this import at the top of the file

Reply:
Done.